### DISCUSSION TITLE
Control Unit compatibility

### DISCUSSION THREAD

**Discussion by betizad** [nliaudat/esp_canbus#20] [Q&A]
Would this also work as a replacement for BG02-E control unit? On the control unit there are no displays but just two knobs to control max humidity and air flow percentage. My goal is to integrate it with homeassitant so I can control it in my way and based on certain scenarios automatically.

**Comment by nliaudat** [comment]
The BG02-E control unit seems to use 24V. 
The canbus channel is on 12V. 
I mean it's not the same control mode.

**Comment by nliaudat** [comment]
Have a look at https://raw.githubusercontent.com/wladwnt/CAN-Gateway/master/Operating_manual.pdf
The v5 board supports the BG02 control unit.
